tasks/test.rake

```diff
@@ -7,7 +7,6 @@ task :setup, :parallel do |t, opts|
     system 'mkdir -p spec/rails'
     args = %w[
       -m\ spec/support/rails_template.rb
```

Comment:
Sorry I miss read the code, I only looked on the code formating, with removing the `\`